modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -3686,10 +3686,13 @@ type GroupEdit {
   """
   editType: EditType!
 
+  "Assocated program"
+  program: Program!
+
   """
-  Edited object
+  Edited group, or null if it's the top-level group.
   """
-  value: Group!
```

Comment:
I don't think this has to be nullable. If the root group is updated it's fine if just the elements inside it are updated (with a new index)

Reply:
I think for consistency it needs to be nullable, but you can ignore this case on your end if you like.